lua/lvim/utils/hooks.lua

```diff
@@ -12,25 +12,22 @@ function M.run_pre_reload()
 end
 
 function M.run_on_packer_complete()
-  vim.schedule(function()
-    if not in_headless then
-      -- colorscheme must get called after plugins are loaded or it will break new installs.
-      vim.g.colors_name = lvim.colorscheme
-      vim.cmd("colorscheme " .. lvim.colorscheme)
```

Comment:
I think we should keep these since they always cause issues when we remove them

Reply:
@abzcoding, btw, any idea what's wrong with stylua? I'm not getting any errors locally even after updating to v0.12.5

Reply:
well, you've solved the stylua 😅 
I'm not sure about the colorscheme thingy, whenever we remove any of them half of the users complain 😄

Reply:
I'll try to put it back and see if there are any (serious) errors. 

It looks like there was a new release of stylua just yesterday, but it didn't quite make it to the AUR  😄😢

Reply:
had to change back the `schedule_wrap()` call to `schedule()` since it only seemed to work in tests. 

I think this may require a few testers to see if they're happy with this workflow.